registry/hardhat.config.ts

```diff
@@ -0,0 +1,65 @@
+import { HardhatUserConfig, extendEnvironment } from "hardhat/config";
```

Comment:
This is the subset of the config from evm-tableland that's needed for the local chain to start and deploy

Reply:
will this get replaced by the config from evm-tableland? or a permanent thing?

Reply:
This would be permanent since hardhat needs the file to exist in this package.